dds_cli/__main__.py

```diff
@@ -468,7 +474,8 @@ def logout(click_ctx):
         with dds_cli.auth.Auth(
             authenticate=False, token_path=click_ctx.get("TOKEN_PATH")
         ) as authenticator:
-            authenticator.logout()
+            logout_status = authenticator.logout()
+            dds_cli.message_helper.MessageHelper().logout_message(logout_status)
```

Comment:
I like the `MessageHelper` thing, but I'm wondering if it's necessary -- will the gui need to call this specific logout message method too? If not then I'm wondering if we shouldn't just handle the logic regarding the messages right here. No real strong opinions atm just trying to get a sense of it  alll